ql/src/java/org/apache/hadoop/hive/ql/udf/UDFLike.java

```diff
@@ -61,7 +61,10 @@ private enum PatternType {
   public UDFLike() {
   }
 
-  public static String likePatternToRegExp(String likePattern) {
+  public static String likePatternToRegExp(String likePattern, boolean quote) {
```

Comment:
since it's used in multiple places maybe you can keep the default impl with 1 param
````
public static String likePatternToRegExp(likePattern){
  return likePatternToRegExp(likePattern, true);
}
````

Reply:
Addressed.